core/templates/dev/head/components/summary_tile/ExplorationSummaryTileMobileDirective.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for an exploration summary tile.
+ */
+
+oppia.directive('explorationSummaryTileMobile', [function() {
```

Comment:
That's a fair point, but there is so much code being reused/copied that duplicating doesn't feel great either. I would say there are many more similarities than differences, and the underlying concept is the same, so we should probably just have one thing. What do you think?

If so, I think having one thing means adding a showButtonAtBottom option that hides the stats and shows the "play exploration" button (which seems reasonably straightforward). I'm less certain about the dimensions -- does the preview card on this page on mobile match the dimensions of other summary tiles on mobile, e.g. in the library (or should it)? If the two sizes should be different, we'll need another option here too -- isCollectionPreviewTile or similar, perhaps.

Are there other differences?